appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -778,7 +779,7 @@ Blockly.ReplMgr.startAdbDevice = function(rs, usb) {
     } else {
         message = Blockly.Msg.REPL_STARTING_EMULATOR;
     }
-    progdialog = new Blockly.Util.Dialog(Blockly.Msg.REPL_CONNECTING, message, Blockly.Msg.REPL_CANCEL, null, 0, function() {
+    progdialog = new Blockly.Util.Dialog(Blockly.Msg.REPL_CONNECTING, message, Blockly.Msg.REPL_CANCEL, true, null, 0, function() {
```

Comment:
What is the rationale for marking the canceling emulator startup as destructive?

Reply:
Yeah, it should be a warning really, not a red flag... I can change it back, not a problem.